README.md

```diff
@@ -73,7 +81,15 @@ Do not manually edit either of these files. If you need to override styles from
 
 #### Updating `orbit.css`
 
-WIP
+This process is manual and messy. The steps below might not be complete. Go slow and update this README is you find any gaps. Good luck!
```

Comment:
Hah, reassuring. 

Maybe just say:
"This process is manual. The steps below might not be complete. Update this README if you find any gaps."

Reply:
haha, your edits are much better. Updating. Thanks!